profiles/pci/handhelds/profiles.toml

```
@@ -110,6 +110,20 @@ post_install = """
     for mask in ${maskservices[@]}; do
         systemctl mask "${mask}.service"
     done
+    echo "Installing audio profile..."
+    product_name="$(cat /sys/devices/virtual/dmi/id/product_name)"
+    mkdir -p /usr/share/pipewire/pipewire.conf.d /usr/share/wireplumber/wireplumber.conf.d/
+    ln -s /usr/share/cachyos-handheld/rog-ally/pipewire/filter-chain.conf \
+    /usr/share/pipewire/pipewire.conf.d
+    if [[ "$product_name" =~ RC71L ]]; then
+        ln -s /usr/share/cachyos-handheld/rog-ally/wireplumber/alsa-card0.conf \
+        /usr/share/wireplumber/wireplumber.conf.d
+    else
+        ln -s /usr/share/cachyos-handheld/rog-ally/wireplumber/alsa-card0-x.conf \
+        /usr/share/wireplumber/wireplumber.conf.d
+    fi
+    ln -s /usr/share/cachyos-handheld/common/wireplumber/alsa-card1.conf \
+    /usr/share/wireplumber/wireplumber.conf.d
```

Nit: it's better to add a space when splitting the line so that there is no confusion.
```suggestion
        /usr/share/wireplumber/wireplumber.conf.d
```